core/docker-compose.yml

```diff
@@ -67,6 +67,7 @@ services:
     - IEXEC_RESULT_REPOSITORY_PROTOCOL=https
     - IEXEC_PLATFORM_REGISTRY=http://platform-registry:8888
     - IEXEC_PLATFORM_REGISTRY_STACK=$CHAIN_ID
+    - MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE=health,prometheus
```

Comment:
Should we comment this to avoid enabling Prometheus metrics if the user does not want/need them ?

Reply:
The README mentions the URLs /actuator/health and /metrics, I think we should let it as is.

Reply:
au temps pour moi, c'est /actuator/health l'endpoint prometheus... je le vire